wgpu-types/src/lib.rs

```diff
@@ -3706,6 +3710,16 @@ pub struct ImageDataLayout {
 pub enum BufferBindingType {
     /// A buffer for uniform values.
     ///
+    /// Example WGSL syntax:
+    /// ```rust,ignore
+    /// struct Globals {
+    ///     a_uniform: vec2<f32>,
+    ///     another_uniform: vec2<f32>
```

Comment:
```suggestion
    ///     another_uniform: vec2<f32>,
```

for consistency

Reply:
Looks good. I think I did that because I wasn't sure if *naga* (and the WGSL spec?) supported trailing commas in struct declarations (I previously ran into issues with that, I think). But I see it in the spec now: https://gpuweb.github.io/gpuweb/wgsl/#syntax-struct_body_decl.